front/src/pages/OfflineScreenWrapper.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactNode, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+
+interface IOfflineScreenWrapper {
```

Comment:
we don't use interface, we use types to avoid mixing both and simplify dev experience

Reply:
we also usually call them OwnProps (see other components)